Assets/Mirror/Weaver/CompilationFinishedHook.cs

```diff
@@ -13,6 +13,14 @@ namespace Mirror.Weaver
     [InitializeOnLoad]
     public class CompilationFinishedHook
     {
+        public static Action<string> OnWeaverMessage; // delegate for subscription to Weaver debug messages
+        public static Action<string> OnWeaverWarning; // delegate for subscription to Weaver warning messages
+        public static Action<string> OnWeaverError; // delete for subscription to Weaver error messages
+
+        public static bool WeaverDisabled { get; set; } // controls whether we weave any assemblies when CompilationPipeline delegates are invoked
+        public static bool UnityLogDisabled { get; set; } // controls weather Weaver errors are reported direct to the Unity console (tests enable this)
```

Comment:
make this a positive variable such as UnityLogEnabled.

Negative variables are hard to read because you end up with double negatives such as 
```cs
if (!UnityLogDisabled) {...}
```

Reply:
I was lazy and trying to just let it be default value enabled ... I'll fix that